tutorials/privacy/payjoin-samourai wallet/tutorial.yml

```diff
@@ -0,0 +1,14 @@
+builder: Payjoin - Samourai Wallet
+
+tags:
+  - privacy
+  - payjoin
+  - P2EP
```

Comment:
Ici tu as mis P2EP. Est-ce voulu ? ou c'est simplement P2P

Reply:
C'est voulu ;) P2EP = Pay-to-End-Point